### PR TITLE
[PR] Implementing logout.

### LINKED ISSUE #29: what about logout?
Hello... great lib btw.

what about logout? there is some gotcha to block an ad user and the user logout the phoenix app?

thanks

### PR DESCRIPTION
**Pull request by LuchoTurtle**
closes #29 

This PR is very much in progress but will detail the process to add a `logout` https://github.com/dwyl/elixir-auth-microsoft/issues/29 function that will:

- Clear the cache/local storage/cookies with user information on the browser.
- Clear the session on the identity server in Azure.

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/logout.md

Also adds a section to clarify what to do if the person faces the following error:

image

`ErrorInsufficientPermissionsInAccessToken`
`"Exception of type 'Microsoft.Fast.Profile.Core.Exception.ProfileAccessDeniedException' was thrown."}`

### CODE REVIEW

**Comment by LuchoTurtle** [comment]
@nelsonic this should be easily mergeable 👌

File: README.md [hunk 1]
```diff
@@ -239,7 +249,7 @@ defmodule AppWeb.MicrosoftAuthController do
   use AppWeb, :controller
 
   @doc """
-  `index/2` handles the callback from Google Auth API redirect.
+  `index/2` handles the callback from Microsoft Auth API redirect.
```
Comment (nelsonic) [inline README.md:252]: good catch. thanks for updating.

**Review by nelsonic — APPROVED** [review]
@LuchoTurtle looks great! 🎉 
When you're back at your desk, please publish the new version to Hex.pm so people can try it. 👌

**Comment by LuchoTurtle** [comment]
It's done publishing -> https://hex.pm/packages/elixir_auth_microsoft

**Comment by nelsonic** [comment]
Nice. 👌

**Pull request merged**